Vision/proposal.md

```diff
@@ -0,0 +1,62 @@
+# The Mission of the W3C
+
+The W3C leads the community in defining a World Wide Web that is inclusive and respectful of its users, 
+to empower an ethical, equitable, informed, and interconnected society.
+
+## **Introduction**
+### The World Wide Web
+
+The World Wide Web was originally conceived as a tool for sharing information. 
+It has evolved rapidly into a fundamental part of humanity, 
+sparking major social change by providing access to 
```

Comment:
```suggestion
sparking major social change by providing and expanding access to 
```
(all of these things were already doable before the Web)

Reply:
fixed in #111.